scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -437,6 +440,11 @@ def test_compare_reports(pkg_dir: str, changelog: bool, source_report: str = "st
         stable = json.load(fd)
     with open(os.path.join(pkg_dir, target_report), "r") as fd:
         current = json.load(fd)
+
+    if "mgmt" in package_name:
```

Comment:
Only applying doing this for mgmt plane packages since I dont believe we do this for data-plane anywhere and it's just extra processing that we dont need for data-plane

Reply:
Let me make a test locally with this PR and will update later.

Reply:
done